packages/snaps-sdk/CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [6.1.1]
+### Fixed
+- Bump `@metamask/providers` from `17.0.0` to `^17.1.2` ([#2598](https://github.com/metamask/snaps/pull/2598))
+  - Resolves compatibility issue with tools that don't support package subpath exports (e.g. Browserify).
```

Comment:
```suggestion
```

Reply:
Got it. Should I recategorize to "changed" as well or leave as "fixed"?

Reply:
I guess changed makes more sense.